sakaar/urls.py

```diff
@@ -14,8 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('api/', include('heroes.urls')),
```

Comment:
With reference to previous comment, here you could have:
```suggestion
    path('api/heroes/', include('heroes.urls')),
```